<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QMainWindow" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">url:/newPrefix/배경.png</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="idLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>아이디</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>비밀번호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이름</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="phLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;</string>
    </property>
    <property name="text">
     <string>숫자만 입력</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전화번호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="registButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>680</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 2</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
background-color: rgb(101, 240, 224);
}

QPushButton:hover {
    
	background-color: rgb(255, 255, 158);
}
</string>
    </property>
    <property name="text">
     <string>가입하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="idcheckBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 158);
}

QPushButton:hover {
	background-color: rgb(101, 240, 224);
}
</string>
    </property>
    <property name="text">
     <string>중복 확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phcheckBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 255, 158);
}

QPushButton:hover {
	background-color: rgb(101, 240, 224);
}
</string>
    </property>
    <property name="text">
     <string>중복 확인</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 2</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 255);
color: rgb(255, 141, 96);
text-shadow: 1px 1px 2px black</string>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
background-color: rgba(255, 69, 190, 80);
color: rgb(4, 4, 4);</string>
    </property>
    <property name="text">
     <string>회원가입 하고 점수를 기록해보세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>800</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/회원가입.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>idLine</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pwLine</zorder>
   <zorder>nameLine</zorder>
   <zorder>label_3</zorder>
   <zorder>phLine</zorder>
   <zorder>label_4</zorder>
   <zorder>registButton</zorder>
   <zorder>idcheckBtn</zorder>
   <zorder>phcheckBtn</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
